libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -263,6 +263,14 @@ singleton AP_Baro method get_pressure float
 singleton AP_Baro method get_temperature float
 singleton AP_Baro method get_external_temperature float
 
+include AP_OpticalFlow/OpticalFlow.h
+singleton OpticalFlow alias optical_flow
+singleton OpticalFlow method enabled boolean
```

Comment:
```suggestion
singleton OpticalFlow  depends defined(OPTFLOW) && OPTFLOW == 1
singleton OpticalFlow method enabled boolean
```

Reply:
I don't think this method will work because the definition is in the vehicle code instead of in the HAL_ or libraries layer.

Reply:
maybe it will have to wait for re-work of the define, would be nice to get it working for the build server. Possibly this might work?

`!defined(OPTFLOW)  || (defined(OPTFLOW) && OPTFLOW == 1)`